app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  @@renderer = Redcarpet::Markdown.new(Redcarpet::Render::HTML, autolink: true, tables: true)
+
+  def render_markdown_as_html markup
+    raw @@renderer.render markup
```

Comment:
@nicholasjhenry Good catch. And I'm sorry I overlooked that detail. Fixed using a `push -f` on that PR.

Suggestion: maybe the project could benefit from using a static code analyzer like [rubocop](https://github.com/bbatsov/rubocop)?